aii-core/src/main/eg/aii-shellfe.conf

```diff
@@ -15,3 +15,28 @@
 # The default is profile_ for backward compatibility
 # Comment this out when not prefix is needed
 profile_prefix = profile_
+
+# Directory on TFTP server where PXELINUX configuration files are.
+# Must be an absolute path or none to disable PXELINUX configuration.
+# nbpdir = /osinstall/nbp/pxelinux.cfg
+
+# Directory on TFTP server where Grub2 PXE configuration files are.
+# Must be an absolute path or none to disable Grub2 PXE configuration.
+# Default value is based on 'npbdir' option (used for PXELINUX) if defined, 
+# with last path token replaced by 'grub-efi'. If 'nbpdir' is not defined
+# (or 'none') the default value is '/osinstall/nbp/grub-efi'.
+# nbpdir_grub2 =
+
+
+# Parent directory for the installation kernel/initrd to use with Grub2 EFI.
+# Must be an absolute path (starting with /), relative to TFTP server root directory.
+# Default value is based on 'nbpdir_grub2' with first and last path tokens removed
+# (if 'nbpdir_grub2' is /a/b/c/d, 'grub2_efi_kernel_root' will be '/b/c').
+# grub2_efi_kernel_root =
+
+# Grub2 command to use to load the installation kernel.
+# grub2_efi_linux_cmd = linuxefi
+#
+# Grub2 command to use to load initrd.
+# Default based on 'grub2_efi_linux_cmd' (matching initrd command)
+# grub2_efi_initrd_cmd = initrdefi
```

Comment:
This is intentional (as the initrd command **MUST** match the linux command) and it was a mistake to add it in the default conf file. Removed.